edgelet/edgelet-http/src/version.rs

```diff
@@ -8,7 +8,7 @@ pub enum ApiVersion {
     V2019_10_22,
     V2019_11_05,
     V2020_07_07,
-    V2020_10_10,
```

Comment:
It looks like this version was previously used so we should not remove it, any new version should be in addition